doc/source/static_surfaces.md

```diff
@@ -0,0 +1,81 @@
+(static-surfaces)=
+
+# Static surfaces as error proxy
+
+## The great benefactor of quantitative elevation analysis
+
+Elevation data benefits from an uncommon asset, which is that **large proportions of planetary surface elevations
+usually remain virtually unchanged through time**. Those static surfaces, sometimes also referred to as "stable terrain",
```

Comment:
The geologist in me wants you to add "within human time-scales". Everything moves significantly if you wait long enough!

Reply:
Good point, added!